arestor/cli/commands/user.py

```diff
@@ -89,6 +89,35 @@ def _work(self):
         return users.list_users()
 
 
+class _ShowSecret(cli_base.Command):
+
+    """Show the secret for a given user."""
+
+    def _on_task_done(self, result):
+        """What to execute after successfully finished processing a task."""
+        table = prettytable.PrettyTable(["API Key", "Secret"])
+        if result:
+            table.add_row(result)
+        print(table)
+
+    def setup(self):
+        """Extend the parser configuration in order to expose this command."""
+        parser = self._parser.add_parser(
+            "show", help="Show the secret for a given user.")
+        parser.add_argument(
+            "api_key", help="The client key id.")
+        parser.set_defaults(work=self.run)
+
+    def _work(self):
+        """Return a specific user."""
+        users = arestor_tools.Users()
+        secret = users.get_secret(api_key=self.args.api_key)
+        if secret:
+            return self.args.api_key, secret
+        else:
+            return None
```

Comment:
I replicate the openstack projects behaviour, `glance image-list` when nothing to show a blank table is displayed